specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/securityContacts.json

```diff
@@ -0,0 +1,380 @@
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityContact"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Deletes a security contact data": {
+            "$ref": "./examples/SecurityContacts/DeleteSecurityContact_example.json"
+          }
+        },
+        "tags": [
+          "Security Contacts"
+        ],
+        "description": "Security contact configurations for the subscription",
+        "operationId": "SecurityContacts_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/SecurityContactName"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityContactList": {
+      "type": "object",
+      "description": "List of security contacts response",
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "description": "List of security contacts",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecurityContact"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "SecurityContact": {
+      "type": "object",
+      "description": "Contact details for security issues",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Security contact data",
+          "$ref": "#/definitions/SecurityContactProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecurityContactProperties": {
+      "type": "object",
+      "description": "Describes security contact properties",
+      "properties": {
+        "emails": {
+          "type": "string",
+          "description": "The security contact's emails list. Defines which emails will get notifications in case option is enables. Emails will be provided as string, separated by delimiter ;"
```

Comment:
Why are you using a delimited string and not an array of strings?

Reply:
So we won't break policies that were defined on the old versions (which has one string email per security contact)